SDK_CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Releases
 
+### v0.84.0 - (6/24/2020)
+
+* Updated rebalance data structures
```

Comment:
ah, is this needed to make CI pass?

I was hoping to skip that since the rebalance structures were never released so we didn't need to bump the version.

Reply:
unfortunately it won't allow me to merge. It is not clear why we need to update this text file - we can just generate it always from git commits.